internal/common/rules/globalvariablecache.go

```diff
@@ -0,0 +1,30 @@
+package rules
+
+import "errors"
+
+type VariableValuePair struct {
+	VariableName string
+	Multivalued  bool
+	SingleValue  float64
+	MultiValue   []float64
```

Comment:
wouldn't SingleValue be implied from `Values []float64` having one value?

Reply:
Yeah idk why I even tried this tbh XD, fixed now